package.json

```diff
@@ -99,6 +99,7 @@
     "react-native-vector-icons": "^7.1.0",
     "react-native-webview": "^13.5.1",
     "rimraf": "^3.0.2",
+    "rn-id-blurview": "^1.2.1",
```

Comment:
You will need to add this to the list of native dependencies in the Readme file.